hpOneView/common.py

```diff
@@ -1695,6 +1696,24 @@ def make_alertMap_dict(notes, etag, state='Active', user='None',
     }
 
 
+def make_migration_information(oaIpAddress, oaUsername, oaPassword, vcmUsername, vcmPassword, iloLicenseType='OneView',
```

Comment:
Personally, I would prefer to have the static method on the class VcMigrationManager .

@balestrinc @tiagomtotti @GustavoHennig @bsouza What do you think?